src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -386,7 +388,19 @@ public void write() {
             }
           case SHOULD_ROTATE:
             updateRotationTimers(currentRecord);
-            closeTempFile();
+            try {
+              closeTempFile();
+            } catch (Exception e) {
+              log.error("Failed to close temp file.",e);
```

Comment:
This is a condition that is expected to be recovered from, so the priority should be `warn` instead of `error`.

Reply:
OK